packages/table-dev-app/src/index.scss

```diff
@@ -1,6 +1,7 @@
 // Copyright 2017 Palantir Technologies, Inc. All rights reserved.
 // Licensed under the terms of the LICENSE file distributed with this project.
 
+@import "overrides";
```

Comment:
I think it is better to import customs at the begining and overrides at the end of the other scss imports. Like we do in `core`

Reply:
Done https://github.com/graphext/blueprint/pull/18/commits/38a22b509a106f6a5a7f366b45b409cc603a1010